x11_forwarding/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+# tasks file for x11_forwarding role
+
+- name: Add EPEL repo on CentOS
+  yum:
+    name: epel-release
+    state: present
+  when: ansible_distribution == 'CentOS'
+  tags: [centos, repo]
+
+- name: Install mate desktop on CentOS
+  yum:
+    name: "@mate-desktop"
+    state: present
+  when: ansible_distribution == 'CentOS'
+  tags: [centos, mate]
+
+- name: Check CentOS runlevel
+  command: systemctl get-default
```

Comment:
This will likely break on CentOS 6. Please add a version check to the where clauses, and update the README to specify CentOS 7+.

Reply:
Added support for only CentOS 7